/*
 * DriverPosition â€” driver seat position in vehicle.
 *
 * APK class: vxi.java (AA v16.1)
 * Used in ServiceDiscoveryResponse field 6 (was bool left_hand_drive_vehicle).
 */

syntax="proto3";

package oaa.proto.enums;

message DriverPosition
{
    enum Enum
    {
        LEFT = 0;
        RIGHT = 1;
        CENTER = 2;
        UNKNOWN = 3;
    }
}
